CMakeLists.txt

```diff
@@ -367,28 +367,43 @@ if(WITH_CUDA)
   include("${PCL_SOURCE_DIR}/cmake/pcl_find_cuda.cmake")
 endif()
 
+
+# Find VTK - VTK has to be found before Qt, otherwise it can overwrite Qt variables
+option(WITH_VTK "Build VTK-Visualizations" TRUE)
+if(WITH_VTK AND NOT ANDROID)
+  include("${PCL_SOURCE_DIR}/cmake/pcl_find_vtk.cmake")
+else()
+  set(VTK_FOUND OFF)
+  if(ANDROID)
```

Comment:
This will always emit a warning despite having WITH_VTK off

Reply:
Ah, true. I'll add a condition with WITH_VTK.